kissmetrics-provider/build.gradle

```diff
@@ -52,14 +52,17 @@ android {
 }
 
 dependencies {
-    implementation 'androidx.core:core-ktx:1.6.0'
+    implementation 'androidx.core:core-ktx:1.8.0'
 
     implementation project(path: rootProject.ext.analytics_kit)
     api "org.jetbrains.kotlin:kotlin-stdlib:$rootProject.ext.kotlinVersion"
-    api "androidx.annotation:annotation:1.2.0"
-    compileOnly 'com.kissmetrics.sdk:KISSmetricsSDK:2.3.1'
+    api "androidx.annotation:annotation:$rootProject.ext.annotationsVersion"
+//    compileOnly 'com.kissmetrics.sdk:KISSmetricsSDK:2.3.1' // Not working
```

Comment:
The latest versions of theKISSmetrics SDK were on `jcenter()` which has been sunsetted. As a temporary (I hope :crossed_fingers: ) workaround I forked their SDK repo and made it so Jitpack could build the latest version.

Reply:
That's too bad. 
Is their library deprecated? Or they just haven't gotten around to moving it?

Reply:
It seems like they haven't paid much attention to their SDK if they haven't made the time to migrate to a different public repository. I raised a [PR](https://github.com/kissmetrics/KISSmetrics-Android-SDK/pull/28) in case they want to be able to use Jitpack to build their binaries. :man_shrugging: